bin/build.mjs

```diff
@@ -78,6 +83,8 @@ async function buildPackage(absWorkingDir) {
     loader: { '.frag': 'text', '.vert': 'text' },
     sourcemap: DEV ? 'inline' : false,
     absWorkingDir,
+    outdir: 'lib',
+    tsconfig: path.join(absWorkingDir, 'tsconfig.json'),
```

Comment:
needed for decorators

Reply:
what happens if we don't specify `tsconfig` here? Does it attempt to discover `tsconfig.json` near from its project root instead of the cwd?